src/App.jsx

```diff
@@ -14,70 +16,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {selectedGood === ''
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )
+        : (
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+          <h1 className="title is-flex is-align-items-center">
+            {`${selectedGood} is selected`}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
             <button
-              data-cy="AddButton"
+              onClick={() => setGood('')}
+              data-cy="ClearButton"
               type="button"
-              className="button"
+              className="delete ml-3"
+            />
+          </h1>
+        )}
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              key={good}
+              data-cy="Good"
+              className={cn(
+                { 'has-background-success-light': selectedGood === good },
+              )}
             >
-              +
-            </button>
-          </td>
+              {selectedGood === good
+                ? (
+                  <td>
+                    <button
+                      onClick={() => setGood('')}
```

Comment:
```suggestion
                      onClick={clearGood}
```
Here's an example of code reuse.